TypedSyntax/src/show.jl

```diff
@@ -0,0 +1,165 @@
+function Base.show(io::IO, ::MIME"text/plain", node::TypedSyntaxNode; show_byte_offsets=false)
+    println(io, "line:col│$(show_byte_offsets ? " byte_range  │" : "") tree                                   │ type")
+    JuliaSyntax._show_syntax_node(io, Ref{Union{Nothing,String}}(nothing), node, "", show_byte_offsets)
+end
+
+function JuliaSyntax._show_syntax_node(io, current_filename, node::TypedSyntaxNode, indent, show_byte_offsets)
+    fname = node.source.filename
+    line, col = source_location(node.source, node.position)
+    posstr = "$(lpad(line, 4)):$(rpad(col,3))│"
+    if show_byte_offsets
+        posstr *= "$(lpad(first_byte(node),6)):$(rpad(last_byte(node),6))│"
+    end
+    val = node.val
+    nodestr = haschildren(node) ? "[$(untokenize(head(node)))]" :
+    isa(val, Symbol) ? string(val) : repr(val)
+    treestr = string(indent, nodestr)
+    if node.typ !== nothing
+        treestr = string(rpad(treestr, 40), "│$(node.typ)")
+    end
+    println(io, posstr, treestr)
+    if haschildren(node)
+        new_indent = indent*"  "
+        for n in children(node)
+            JuliaSyntax._show_syntax_node(io, current_filename, n, new_indent, show_byte_offsets)
+        end
+    end
+end
+
+
+function Base.printstyled(io::IO, rootnode::MaybeTypedSyntaxNode;
+                          type_annotations::Bool=true, iswarn::Bool=true, hide_type_stable::Bool=true,
+                          idxend = last_byte(rootnode), kwargs...)
```

Comment:
```suggestion
                          idxend = last_byte(rootnode))
```
Since `kwargs` is unused at all, I think we should remove it since otherwise we will accept whatever keyword argument silently.